.github/workflows/release.yml

```diff
@@ -42,67 +44,73 @@ jobs:
+          echo "Release KLT Cert Manager:     ${{ steps.release.outputs.klt-cert-manager--release_created }}"
+          echo "Release KLT Metrics Operator: ${{ steps.release.outputs.metrics-operator--release_created }}"
+          echo "Anything to release:          ${{ steps.release.outputs.releases_created }}"
+          echo "Paths to be released:         ${{ steps.release.outputs.paths_released }}"
 
       - name: Create build matrix
         id: build-matrix
         uses: actions/github-script@v6
         env:
           RELEASE_KLT: ${{ steps.release.outputs.release_created }}
           RELEASE_CERT_MANAGER: ${{ steps.release.outputs.klt-cert-manager--release_created }}
+          RELEASE_METRICS_OPERATOR: ${{ steps.release.outputs.metrics-operator--release_created }}
           KLT_TAG: ${{ steps.release.outputs.tag_name }}
           CERT_MANAGER_TAG: ${{ steps.release.outputs.klt-cert-manager--tag_name }}
+          METRICS_OPERATOR_TAG: ${{ steps.release.outputs.metrics-operator--tag_name }}
         with:
           script: |
-            const { RELEASE_KLT, RELEASE_CERT_MANAGER, KLT_TAG, CERT_MANAGER_TAG } = process.env
-            const kltMatrix = [
-              {
-                  name: "lifecycle-operator",
-                  folder: "operator/",
-                  tagName: KLT_TAG
-              },
-              {
-                  name: "metrics-operator",
-                  folder: "metrics-operator/",
-                  tagName: KLT_TAG
-              },
-              {
-                  name: "scheduler",
-                  folder: "scheduler/",
-                  tagName: KLT_TAG
-              },
-              {
-                  name: "functions-runtime",
-                  folder: "functions-runtime/",
-                  tagName: KLT_TAG
-              },
-              {
-                  name: "python-runtime",
-                  folder: "python-runtime/",
-                  tagName: KLT_TAG
-              }
-            ]
-
-            const certManagerMatrix = [
-              {
-                  name: "certificate-operator",
-                  folder: "klt-cert-manager/",
-                  tagName: CERT_MANAGER_TAG
-              }
-            ]
-
-            let result = {}
-            if (RELEASE_KLT === "true" && RELEASE_CERT_MANAGER === "true") {
-                result = { config: [...kltMatrix, ...certManagerMatrix]}
-            } else if (RELEASE_KLT === "true") {
-                result = { config: kltMatrix }
-            } else if (RELEASE_CERT_MANAGER === "true") {
-                result = { config: certManagerMatrix }
+            const { RELEASE_KLT, RELEASE_CERT_MANAGER, RELEASE_METRICS_OPERATOR, KLT_TAG, CERT_MANAGER_TAG, METRICS_OPERATOR_TAG } = process.env
+
+            var result = []
+            if (RELEASE_KLT === "true") {
+              result = result.concat([
+                {
```

Comment:
so the ticket on operator monorepo is still needed to separate this guy from scheduler and runtimes? OR should they all stay togheter under lifecycle...

Reply:
I think the scheduler and runtimes need to be part of one piece, as without the scheduler the lifecycle-operator does not actually work and runtimes are part of the Tasks which are executed by lifecycle-operator. So I consider that as one piece

Reply:
Agree than they are lifecycle operator 😸